com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/v1/ApplicationsCloudModelBuilder.java

```diff
@@ -149,6 +164,12 @@ public DeploymentMode getDeploymentMode() {
         return DeploymentMode.SEQUENTIAL;
     }
 
+    private void printDockerImageDetectedMessage(String moduleName) {
+        if (userMessageLogger != null) {
+            userMessageLogger.info(Messages.DOCKER_IMAGE_DETECTED, moduleName);

```

Comment:
Print this in `CreateAppStep` and change the message to `"Creating app \"{0}\" from Docker image \"{1}\"..."`. That way the message would be printed exactly during the application's creation and the information will therefore be grouped in a better way.